pages/index.jsx

```diff
@@ -1,10 +1,20 @@
 var React = require('react');
 
+var Habitat = require('habitat');
+Habitat.load();
```

Comment:
I assumed this index.jsx file was just being run in the pre-compile. And we'd end up with static HTML... but I might have missed something important here.